lib/iwdp.js

```diff
@@ -0,0 +1,78 @@
+import { SubProcess } from 'teen_process';
+import EventEmitter from 'events';
+import { execSync } from 'child_process';
+import Promise from 'bluebird';
+import logger from './logger';
+
+const request = Promise.promisify(require('request'));
+
+const IWDP_CMD = 'ios_webkit_debug_proxy';
+const MAX_RETRIES = 10;
+
+class IWDP extends EventEmitter {
+
+  constructor (udid = null) {
```

Comment:
I still think we ought to have it configurable. I'm sure there is a valid use case for wanting things to run on a specific port.